### DISCUSSION TITLE
Audio Adaptor Board Rev D2

### DISCUSSION THREAD

**Discussion by nekton2** [ElectroTechnique/TSynth-Teensy4.1#128] [T4.1 code development]
Is the new revision D2 usable?

"Rev D2 is functionally the same as Rev D, except the the I2C address selection pads. The I2C address is not configurable on the 20 pin SGTL5000 chip."

**Comment by Electrotechnique** [comment]
Sorry for the late reply. Yes it has to be compatible with Rev D. The pinout is the same:
[https://www.pjrc.com/store/teensy3_audio.html](https://www.pjrc.com/store/teensy3_audio.html)
